deployments/charts/kuma/templates/cp-deployment.yaml

```diff
@@ -47,6 +47,9 @@ spec:
       {{ toYaml .Values.controlPlane.podSecurityContext | trim | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "kuma.name" . }}-control-plane
+      {{- if not .Values.controlPlane.automountServiceAccountToken }}
+      automountServiceAccountToken: {{ .Values.controlPlane.automountServiceAccountToken }}
+      {{- end }}
```

Comment:
Why not always set this?

Reply:
Since automountServiceAccountToken=true by default, I thought it may make sense to enable this code only when the user wants to set it to false. Thoughts? 

PS: I don't mind removing the `if` block completely if that makes more sense..

Reply:
Hmm it's a good point @jakubdyszkiewicz @johnharris85 WDYT?

Reply:
No strong preference, and no functional difference either way as far as I can see. However in general, explicit > implicit so if anything I'd lean towards always setting it.

Reply:
Done, always set.